### ISSUE TITLE
v1.0.2 mac build hangs on splash screen

### ISSUE THREAD

**Issue by chebykin** [trufflesuite/ganache#238]
v1.0.2 macos build hangs on splash screen while Linux build works fine.

**Comment by loktarjugg** [comment]
same here

**Comment by loktarjugg** [comment]
mac os 10.12.6
when I start ganache ,  i got this
![image](https://user-images.githubusercontent.com/3970144/34708224-a2f37140-f54c-11e7-9377-8c26ab68dc6c.png)

**Comment by lichla1988** [comment]
Does nothing on latest mac.  Application opens unable to looking at any options in the Window menu option

![image](https://user-images.githubusercontent.com/17151789/34740350-b64770a0-f544-11e7-8de7-859634176ce7.png)

**Comment by benjamincburns** [comment]
All - my deepest apologies for this. We tested the Windows build on two different machines, but I actually forgot to test the mac build. Sure enough, when we tested it on @tcoulter's mac, it hung.

I'm in the process of rebuilding and republishing, but I want to get confirmation from a few different people that the new build doesn't hang before I post it.

While that's happening we've marked the 1.0.2 release as a prerelease so that it doesn't show up on the truffle website.

Again, I'm very sorry for the false start here. I promise the next build we post won't have this issue!
